static/app/components/panels/panelBody.tsx

```diff
@@ -13,3 +14,20 @@ const PanelBody = styled('div')<BaseProps>`
 `;
 
 export default PanelBody;
+
+export const CollapsePanelBody = styled(motion.div)`
```

Comment:
damn. I kind of don't like that this has a default export and a named export

Reply:
We can only have one default export per file, so we either need to move `CollapsePanelBody` elsewhere, or we need to change `PanelBody` to a named export which is going to cause a lot of breakage. Imo having named and default export in the same file is totally fine, the IDE should be able to figure things out for you.